jis-documentation/src/test/java/pl/edu/uj/JImageStream/tests/predicates/ColorPredicateTest.java

```diff
@@ -0,0 +1,39 @@
+package pl.edu.uj.JImageStream.tests.predicates;
+
+import org.junit.Test;
+import pl.edu.uj.JImageStream.collectors.BufferedImageCollector;
+import pl.edu.uj.JImageStream.filters.edge.RobertsCrossXFilter;
+import pl.edu.uj.JImageStream.filters.noise.SaltAndPepperFilter;
+import pl.edu.uj.JImageStream.predicates.ColorPredicate;
+import pl.edu.uj.JImageStream.predicates.ThresholdPredicate;
+import pl.edu.uj.JImageStream.tests.filters.AbstractBaseTest;
+
+import javax.imageio.ImageIO;
+import java.awt.Color;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Created by pPanek on 2016-11-29.
+ */
+public class ColorPredicateTest extends AbstractBaseTest{
+
+    @Test
+    public void colorPredicateTest() {
+        BufferedImage bufferedImage = streamableImage.parallelStream()
+                .setThreads(50)
+                .bounds(new ThresholdPredicate(155))
+                .apply(new RobertsCrossXFilter())
```

Comment:
why are you using Robers cross filter here and salt and pepper? Could we stick with .bounds(examplePredicate).apply(new RedFilter()) for visibility? Or if RedFilter isn't visible enough, just create sample filter in AbstractBaseTest that will color every pixel red or smth.

public void apply(int x, int y){
setPixel(x, y, new Pixel(255, 0, 0, 255))
}